core/modules/system/system.module

```diff
@@ -2898,6 +2898,16 @@ function _system_rebuild_theme_data() {
 
   // Find theme engines
   $engines = backdrop_system_listing('/^' . BACKDROP_PHP_FUNCTION_PATTERN . '\.engine$/', 'themes/engines');
+  // Allow modules to add further theme engines.
```

Comment:
Not really sure about *expanding* theme engine support. They are on the chopping block. But it might be fine to match Drupal for now and remove it all together later.

Reply:
👍